### PR TITLE
Eq 1461 update enter person 1 details

### PR DESCRIPTION
**Pull request by MIslam92**
### What is the context of this PR?
Piping "address_line_1" into question 

Card: https://trello.com/c/XW1OfqKO/1461-update-enter-person-1-details-s

### How to review 
Run Census Household and check that enter individuals name question now has address piped into it

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -81,6 +81,7 @@ data/cy
 
 keys.yml
 secrets.yml
+package-lock.json
```
Comment (shahi645) [inline .gitignore]: not sure about putting this in gitignore @jonnyshaw89 is this ok?

**Comment by shahi645** [comment]
need to add the extra guidance

**Comment by shahi645** [comment]
needs a yarn format

**Review by shahi645 — APPROVED** [review]
just needs the `yarn format` but LGTM

**Review by charlotte-hirst — APPROVED** [review]

**Comment by shahi645** [comment]
@MIslam92 squishy-squashy mergy-rebasy

**Pull request merged**